app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -399,9 +399,9 @@
     <string name="widgetMode1x1_distance">当前距离</string>     <!-- setting -->
     <string name="widgetMode1x1_apsis">下一个远地点或近地点</string>     <!-- setting -->
 
-    <string name="widgetMode3x1_lightmap_large">Lightmap</string>     <!-- setting -->    <!-- TODO -->
-    <string name="widgetMode3x1_lightmap_medium">Lightmap (medium)</string>     <!-- setting -->    <!-- TODO -->
```

Comment:
The `)` `U+0029` right parenthesis should be in fullwidth `）` `U+FF09` to keep the punctuation consistent throughout the project